### ISSUE TITLE
mimic3 working?

### ISSUE THREAD

**Issue by snwfdhmp** [themanyone/whisper_dictation#9]
Awesome project !

I've seen in your README that you're using mimic3. Did you get it to work ? Is it working currently ?

I'm desperate of finding open source TTS tech for personal use (read articles out loud) but no success yet.

Would appreciate if you can point me to the right direction.

**Comment by themanyone** [comment]
Yes. It's working great. I just committed a [pull rquest to make it work with Python 3.13](https://github.com/MycroftAI/mimic3/issues/61)

**Comment by snwfdhmp** [comment]
Thanks a lot, i will check it out

**Comment by themanyone** [comment]
Maintaining a fork of mimic3 here https://github.com/themanyone/mimic3
